Jenkinsfile

```diff
@@ -1,74 +0,0 @@
-// Nasty use of conditionals to make this work
-// https://issues.jenkins-ci.org/browse/JENKINS-42643
-String upstreamproject = BRANCH_NAME == "master" ? "core/inmanta-core/master" : ""
-
-pipeline {
-    agent any
-
-    options{
-        checkoutToSubdirectory('pytest')
-        disableConcurrentBuilds()
-    }
-
-    triggers {
-        upstream upstreamproject
-    }
```

Comment:
In the new implementation we lose this trigger.

Reply:
Fixed in: https://github.com/inmanta/irt/pull/289